Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_tbag.nut

```diff
@@ -0,0 +1,397 @@
+global function GamemodeTbag_Init
+global function CreateBattery
+
+struct {
+	table<entity, float> players // needed for detecting if tbag grace period exceeded 3 seconds

```

Comment:
these could probably be structs rather than individual tables per property, unless i'm misunderstanding this

Reply:
Well, this gamemode was one done like all the way back 6 months ago when I was fairly new to scripting.

AFAIK now, I could totally use a struct for each individual player for more clarity but is there any downsides to the current tables method?

Reply:
Taking a look again at this code, I honestly recommend keeping it this way because it would involve having to rewrite the entire thing from scratch.